Source/WebInspectorUI/UserInterface/Views/GeneralStyleDetailsSidebarPanel.js

```diff
@@ -86,7 +93,11 @@ WI.GeneralStyleDetailsSidebarPanel = class GeneralStyleDetailsSidebarPanel exten
         if (!domNode || domNode.destroyed)
             return;
 
-        this.contentView.element.scrollTop = 0;
+        if (!this._preserveScroll)
+            this.contentView.element.scrollTop = 0;
+        else
+            this._preserveScroll = false;
```

Comment:
i wonder if we actually even need this anymore.  seems like a legacy behavior back from when the pseudo-class toggles were "above" the main content of the Styles sidebar panel

Reply:
If this is removed, the scroll position is saved when the user changes tabs (or switches to another sidebar pane) and then returns. Personally I would find this an improvement, so I'd be happy to make that change, but others may disagree.

Reply:
yeah im definitely in favor of that.  im pretty sure that was the behavior in the past.  not sure when that changed